### PR TITLE
move sig-cli gke serial tests into their own job

### PR DESCRIPTION
**Pull request by pwittrock**

### CODE REVIEW

File: jobs/config.json [hunk 1]
```diff
@@ -6601,6 +6601,24 @@
       "UNKNOWN"
     ]
   },
+  "ci-kubernetes-e2e-gke-gci-serial-sig-cli": {
+    "args": [
+      "--check-leaked-resources",
+      "--env-file=jobs/platform/gke.env",
+      "--env-file=jobs/ci-kubernetes-e2e-gke-serial-sig-cli.env",
```
Comment (krzyzacy) [inline jobs/config.json]: the env file should be `ci-kubernetes-e2e-gke-gci-serial-sig-cli.env`

**Comment by pwittrock** [comment]
Woot

**Comment by krzyzacy** [comment]
huh?

**Comment by fejta** [comment]
I think Phil is excited for sig-cli to have their own tests :smiling_imp:

**Pull request merged**